spec/services/support_interface/applications_filter_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe SupportInterface::ApplicationsFilter do
+  describe '#filter_records' do
+    it "allows searching by a string" do
+      application_form_in = create(:application_form, first_name: "Foo", last_name: "Bar")
+      application_form_out = create(:application_form, first_name: "Not", last_name: "Bar")
+
+      applications_filter = SupportInterface::ApplicationsFilter.new(params: { q: "Foo" })
+
+      expect(applications_filter.filter_records(ApplicationForm)).to match_array([application_form_in])
+    end
+
+    it "allows searching by a fuzzy matching string" do
+      application_form_in = create(:application_form, first_name: "Foo Baz", last_name: "Bar")
+      application_form_out = create(:application_form, first_name: "Not", last_name: "Bar")
+
+      applications_filter = SupportInterface::ApplicationsFilter.new(params: { q: "Foo Bar" })
+
+      expect(applications_filter.filter_records(ApplicationForm)).to match_array([application_form_in])
+    end
+
+    it "allows searching by a candidate email" do
+      application_form_in = create(:application_form, first_name: "Foo Baz", last_name: "Bar")
```

Comment:
Any need for these `first_name` and `last_name` here?

Reply:
Good shout, I've updated the tests.